app/user.go

```diff
@@ -1066,6 +1066,19 @@ func (a *App) UpdateUser(user *model.User, sendNotifications bool) (*model.User,
 	return rusers[0], nil
 }
 
+func (a *App) UpdateUserActive(userId string, active bool) *model.AppError {
```

Comment:
Wondering if we should just inline this into `PluginAPI.UpdateUserActive` instead of exposing a new app method that's currently only used in the plugin layer. It avoids a layer of indirection, keeping the changes simpler. (It's always easy to reintroduce if we end up using it in more places.)

Reply:
I am fine either way. A lot of other methods were using the same pattern, hence did not want to break the pattern. I can inline it if you feel so.

Reply:
@lieut-data We have used the pattern of adding new methods to the App layer in most of the other plugin API PRs. But not in all. There is also some duplicate code with the REST API.

I think there is a decision to be made how we structure the code base. But this is out of the scope of this PR. 3/5

Reply:
@lieut-data I am assuming, the current code is fine for now or should I inline the code?

Reply:
Fine to leave as-is :)